src/Main.cpp

```diff
@@ -321,6 +392,8 @@ DevicePort portFromString(string const& arg) {
     }
 }
 
+#define CHRONY_SOCK_MAGIC 0x534f434b
```

Comment:
I dont see you using this macro anywhere (it is already defined in chrony.cpp which is the only place that seems to need it). It seems its fine remove this, unless I am getting confused by how macros are included from other source files